compose/install/index.md

```diff
@@ -0,0 +1,39 @@
+---
+description: How to install Docker Compose
+keywords: compose, orchestration, install, installation, docker, documentation
+title: Install Docker Compose
+toc_max: 2
+---
+
+This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.
+
+
+## Prerequisites
```

Comment:
Wonder if you need these prereqs here and repeated on the compose-plugin page? It implies that these are prereqs for Docker Desktop as well.

Reply:
OK I will be changing this to Compose Prerequisites to clarify.